ci/test-elk-flow-collector.sh

```diff
@@ -4,18 +4,65 @@ LOGSTASH_PORT=4739
 LOGSTASH_IP="0.0.0.0"
 LOGSTASH_PROTOCOL="udp"
 FAILURE=false
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GIT_CHECKOUT_DIR=${THIS_DIR}/.
```

Comment:
does that work for Jenkins?

Reply:
It doesn't seem to work but `GIT_CHECKOUT_DIR=${THIS_DIR}/..` works. I have verified on Jenkins.
As for Jenkins, there is a env variable `WORKSPACE` showing path of repo cloned by Jenkins: `$HOME/workspace/jenkins-job-name`

Reply:
I agree that it should be `GIT_CHECKOUT_DIR=${THIS_DIR}/..`. It seems that `GIT_CHECKOUT_DIR=${THIS_DIR}/.` would not work locally either, since the root of the repo is one directory up compared to the directory which contains the script. This is what @Dyanngg used in other CI scripts:
```
# ensures that the script can be run from anywhere
THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
GIT_CHECKOUT_DIR=${THIS_DIR}/..
pushd "$THIS_DIR" > /dev/null
```

Reply:
Confirmed that the current code does not work locally:
```bash
abas-a01:antrea abas$ ./ci/test-elk-flow-collector.sh
=== Starting Antrea ===
error: the path "/Users/abas/vmware/antrea/ci/./build/yamls/antrea.yml" does not exist
```

Reply:
Thanks @lzhecheng and @antoninbas for testing. I was in the wrong directory when I was testing. Thanks for pointing this out. Fixed.